kitty/launch.py

```diff
@@ -81,7 +81,11 @@ def options_spec() -> str:
 --copy-env
 type=bool-set
 Copy the environment variables from the currently active window into the
-newly launched child process.
+newly launched child process. It will not copy environment variables that have
+been :code:`export`ed since the :code:`export` command does not change the environment
```

Comment:
Isn't it a bit misleading to say "variables that have been:`export`ed? It makes it sound like variables you set with export (`export TEST=test`) doesn't work, while variables you just set (`TEST=test`) works, while in reality none of them work, only the variables the shell had when it started?

Reply:
This is a given that `A=B` variables that are not exported are not available to child processes.